TIGLViewer/src/modificators/ModificatorSectionsWidget.h

```diff
@@ -42,11 +77,11 @@ public slots:
     explicit ModificatorSectionsWidget(QWidget* parent = nullptr);
     ~ModificatorSectionsWidget();
 
-    void setCreateConnectedElementI(tigl::CreateConnectedElementI& elementI);
+    void setCreateConnectedElement(Ui::ElementModificatorInterface& element);
 
 private:
     Ui::ModificatorSectionsWidget* ui;
-    tigl::CreateConnectedElementI* createConnectedElementI;
+    Ui::ElementModificatorInterface* createConnectedElement;
```

Comment:
I believe with the current changes, we should store by value here. The `ElementModificatorInterface` goes out of scope at the end of the function where it is created and the pointer can be dangling. In addition, the `ElementModificatorInterface` already stores the fuselage or wing by reference.